inference-engine/src/vpu/graph_transformer/src/stages/rnn.cpp

```diff
@@ -138,6 +143,18 @@ void FrontEnd::parseRNN(const Model& model, const ie::CNNLayerPtr& _layer, const
 
     size_t stateSize = outputs[0]->desc().totalDimSize() / nCells / nBatches;
     size_t cellStateSize = inputs[2]->desc().totalDimSize() / nBatches;
+
+    if (outputs.size() == 3) {
```

Comment:
why are you trying to calculate this parameter by this formula?  stateSize should be the same.
It seems in DeepSpeech nCells and nBatches are swaped.
You can add:
`
    size_t nCells = inputs[0]->desc().dim(Dim::C);
    size_t nBatches = inputs[0]->desc().dim(Dim::H);
`
as a temporary solution when number of outputs == 3.